src/index.js

```diff
@@ -4,6 +4,7 @@ import { createPopup } from './popup'
 import { createTouchPopup } from './touchPopup'
 import { renderPreview, renderLoading, renderError, renderDisambiguation, renderOffline } from './preview'
 import { getWikipediaAttrFromUrl, buildWikipediaUrl, isTouch, getDir, isOnline, version, getAnalyticsQueryParam } from './utils'
+import '../style/link.css'
```

Comment:
hmm i'm not sure whether this belongs here, we used to have the linkStyle.js as the external script, and include the css seperately.

Reply:
That's right, in the webpack config we were setting up two entry points like this:

```
const config = {
  entry: {
    'wikipedia-preview': './src/index.js',
    'default-link-style': './src/linkStyle.js'
  }
...
```

So the idea of including link.css in index.js was to just have one entry point in our new vite config (and therefore only load wikipedia-preview.css for all styling). But if it's better to have multiple entry points we can look into setting up something equivalent in vite. Do you remember if there was a reason we had this css separate?

Reply:
the link css is not part of the wikipedia preview library, it's just a plain link css for the demo website, also use as default link style in wordpress, anyone can replace this css, docs is here
https://github.com/wikimedia/wikipedia-preview#wmf-wp-with-preview

Reply:
wait, this may not be a bad idea overall, in many cases we recommend to [include the link css file](https://github.com/wikimedia/wikipedia-preview#wmf-wp-with-preview), if we include it here then we only need to explain how to overwrite the default link style.

_note: the linkStyle.js is just a simple css and doesn't have any bundle size issue._

Reply:
Okay that makes sense, so that could be part of the documentation update we have in TODO list then

Reply:
https://github.com/wikimedia/wikipedia-preview/pull/177/commits/4e5fd59be3b13e47b343c29b1a06a61d60830e7d